app/views/admins/stats.html.haml

```diff
@@ -7,7 +7,7 @@
 
       .pull-right
         = form_tag('/admins/stats', :method => 'get', class: 'form-inline') do
-          %select{:name => 'range'}
+          %select.form-control{:name => 'range'}
```

Comment:
You only changed to single quotes to double quotes but didn't change the hash syntax so the comment is still valid.